samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Droid/Uno.Toolkit.Samples.Droid.csproj

```diff
@@ -32,6 +32,7 @@
     <WarningLevel>4</WarningLevel>
     <AndroidUseSharedRuntime>True</AndroidUseSharedRuntime>
     <AndroidLinkMode>None</AndroidLinkMode>
+    <AndroidSupportedAbis>armeabi-v7a;x86;x86_64;arm64-v8a</AndroidSupportedAbis>
```

Comment:
You can probably skip x86 to speed up the build

Reply:
@jeromelaban I added x86_64 because otherwise the app wasn't lunching on the emulator dor the UI Tests. Looking at it again now, I see the test scripts are using x86_64, example here: https://github.com/unoplatform/uno.toolkit.ui/blob/0e41b1d92b50f47663d16008259ab6cb0709edd3/build/workflow/scripts/android-uitest-run.sh#L44

Should I just replace those lines with armeabi-v7a;x86 or arm64-v8a?

Reply:
oh sorry, did you mean keep x86_64 and just remove x86? Then yeah that makes sense 👍